network-api/networkapi/tests.py

```diff
@@ -141,6 +148,15 @@ def test_PNI_homepage_redirect_to_foundation_site(self):
             fetch_redirect_response=False
         )
 
+    def tearDown(self):
+        # Re-instante localhost as the default site
+        self.original_default_site.is_default_site = True
+        self.original_default_site.save()
+
+        # Remove the Site Factories
+        self.default_site.delete()
+        self.secondary_site.delete()
```

Comment:
And when we're done with this class we'll re-instate `localhost` as the "default" site, and clean up the other sites we generated.